pandas_gbq/gbq.py

```diff
@@ -421,6 +439,17 @@ def sizeof_fmt(num, suffix='B'):
             num /= 1024.0
         return fmt % (num, 'Y', suffix)
 
+    def query_price_for(self, bytes_num):
```

Comment:
Could we rename `query_price_for` to something more descriptive like `query_price_from_byte_count` or similar?